WebUI/src/views/Create.vue

```diff
@@ -105,6 +107,10 @@ const emits = defineEmits<{
 async function generateImage() {
   await ensureModelsAreAvailable();
   reset();
+  const inferenceBackendService: BackendServiceName = imageGeneration.backend === 'comfyui' ? "comfyui-backend" : "ai-backend"
+  await globalSetup.resetLastUsedInferenceBackend(inferenceBackendService)
```

Comment:
This seems like it would reset the currently selected backend. Wasn't the idea to reset the backend that is currently not in use?

Reply:
Ah I see... yes this should work. Maybe the function name is a bit misleading given that the current backend is passed as an argument but the focus is on resetting another backend. But idk how to do it in a better way...